src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4269,16 +4270,42 @@ private string GetCustomMarkupExtensionValue(XamlMemberDefinition member)
 				})
 				.JoinBy(", ");
 
-			// Get the full globalized namespaces for the custom markup extension and also for IMarkupExtensionOverrides
+			// Get the full globalized names
+			var globalized = new
+			{
+				MarkupType = GetGlobalizedTypeName(markupType.GetFullName()!),
+				IMarkupExtensionOverrides = GetGlobalizedTypeName(XamlConstants.Types.IMarkupExtensionOverrides),
+				XamlServiceProviderContext = GetGlobalizedTypeName(XamlConstants.Types.XamlServiceProviderContext),
+				ProvideValueTargetProperty = GetGlobalizedTypeName(XamlConstants.Types.ProvideValueTargetProperty),
+				PvtpDeclaringType = GetGlobalizedTypeName(member.Member.DeclaringType),
+				PvtpType = GetGlobalizedTypeName(member.Member.Type),
+			};
```

Comment:
I wonder if we should be resolving some of this in the constructor, as it's constant. It may not be called that often, though, so the tradeoff is not evident.

Reply:
yeah, i dont think custom markup is a hot path
plus, it is just a few string manipulations
and, we are talking about compile time here (not that it doesnt matter, but just less)